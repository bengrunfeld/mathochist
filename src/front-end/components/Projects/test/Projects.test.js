import React from 'react'
import Enzyme, { shallow, mount, render } from 'enzyme'
import { MemoryRouter } from 'react-router-dom'

import Projects from '../Projects'


describe('<Projects />', () => {
  test('matches the snapshot', () => {
    const tree = mount(<Projects />)
    expect(tree).toMatchSnapshot()
  })
})
