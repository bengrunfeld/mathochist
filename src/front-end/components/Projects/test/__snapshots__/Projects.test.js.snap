// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Projects /> matches the snapshot 1`] = `
<Projects>
  <div>
    <h1>
      Projects
    </h1>
  </div>
</Projects>
`;
